nuxeo-webdav/src/main/java/org/nuxeo/ecm/webdav/resource/ExistingResource.java

```diff
@@ -105,6 +111,42 @@ public Response delete() {
             return Response.status(423).build();
         }
 
+        Serializable dcSource = doc.getPropertyValue("dc:source");
```

Comment:
Do we need to move stuff in order to perform this deletion ?
For instance, what the future of new "old" document moved to a temporary location ? What the future of "old" document just restored ? Don't we want to delete both documents in this case ?

Reply:
I'm not sure. Could you elaborate more on the use-case scenario ?

Reply:
Seen on slack